packages/perspective-viewer/src/js/utils.js

```diff
@@ -7,6 +7,33 @@
  *
  */
 
+/******************************************************************************
+ *
+ * Drag & Drop Utils
+ *
+ */
+
+global.dragEnter = function dragEnter(ev) {
```

Comment:
While we're here can remove these global event handlers which pollute the global namesapce and change them to be event Listeners on the elements when they're created in the element?

Reply:
The whole drag-and-drop implementation definitely needs a rework - I'm for refactoring the implementation but I'll defer to @texodus on whether it should be part of this PR or a different one.

Reply:
This should be fixed but I agree this PR is not about cleaning up drag & drop generally - I've added a task to the project tracker to tackle this in a future sprint.